### PR TITLE
Update script for external pages

### PR DESCRIPTION
**Pull request by zwalchuk**
Minor fixes to the script for external pages.

Change-type: patch

### CODE REVIEW

File: pages/reference/cli.md [hunk 1]
```diff
@@ -81,76 +113,76 @@ environment variable (in the same standard URL format).
...
+	- [deploy &#60;appName&#62; [image]](#deploy-appname--image)
 
 - Utilities
 
 	- [util available-drives](#util-available-drives)
 
 # Application
 
-## app create &#60;name&#62;
+## <a name="#app-create-name"></a>app create &#60;name&#62;
```
Comment (pimterry) [inline pages/reference/cli.md]: Looks like this is built from the previous CLI docs, not the latest, is that right?

Comment (zwalchuk) [reply pages/reference/cli.md]: @pimterry yes, I'll rebase this with the latest changes.

**Comment by pimterry** [comment]
@zwalchuk looks like this now doesn't have any of the CLI docs changes at all - is that right?

**Comment by zwalchuk** [comment]
@pimterry right again, I thought I had the new changes in master already but it was still the one from a month ago. Should be all correct now.

**Pull request merged**